src/index.html

```diff
@@ -9,10 +9,57 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="style.css"
```

Comment:
The `href` attribute currently links to a single `style.css` file. According to the task requirements, each BEM block should have its own separate CSS file. Please create individual CSS files for each block and update this link accordingly.